gradio/interface.py

```diff
@@ -130,6 +130,22 @@ def _infer_model_type(model):
 
         raise ValueError("model_type could not be inferred, please specify parameter `model_type`")
 
+
+    @staticmethod
+    def update_config_file(self, output_directory):
+        networking.set_interface_types_in_config_file(
+            output_directory,
+            self.input_interface.__class__.__name__.lower(),
+            self.output_interface.__class__.__name__.lower(),
+        )
+
+        if self.input_interface.__class__.__name__.lower() == "sketchpad" or self.input_interface.__class__.__name__.lower() == "textbox":
```

Comment:
Better to define a constant list in the inputs.py file, like 
`INTERFACES_SUPPORTING_SAMPLE_INPUTS = ["sketchpad", "textbox"]`

And then the check will be `if self.input_interface.__class__.__name__.lower() in inputs.INTERFACES_SUPPORTING_SAMPLE_INPUTS`

Even better would be to do a check to see if the interface contains an `get_sample_inputs()` method: https://stackoverflow.com/questions/5268404/what-is-the-fastest-way-to-check-if-a-class-has-a-function-defined